kuma/wiki/models.py

```diff
@@ -1431,6 +1432,11 @@ def get_full_url(self):
     def is_experiment(self):
         return self.slug.startswith(EXPERIMENT_TITLE_PREFIX)
 
+    @property
+    def has_legacy_namespace(self):
+        namespace, separator, _ = self.slug.partition(':')
```

Comment:
Today I learned about [partition](https://docs.python.org/2/library/stdtypes.html#str.partition). I usually use ``namespace = self.slug.split(':', 1)[0]``, but ``partition`` is a good way to both split the string and detect that the partition character is in the string.